dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -112,33 +112,57 @@ internal class RumSessionScope(
     // endregion
 
     // region Internal
-    @Suppress("ComplexCondition")
+
     private fun handleOrphanEvent(
+        event: RumRawEvent,
+        writer: DataWriter<Any>
+    ) {
+        val durationNs = durationSinceLaunchNs(event)
+        if (applicationDisplayed || durationNs > APPLICATION_START_MAX_DURATION_NS) {
+            handleBackgroundEvent(event, writer)
+        } else {
+            handleAppLaunchEvent(event, writer)
+        }
```

Comment:
With no other condition, doesn't it mean we will be starting "ApplicationLaunch" view for eventual app background wake-ups? The reason why we introduced [`trackBackgroundRumEvents()`](https://github.com/DataDog/dd-sdk-android/blob/5ad87041b5e094e5d3d8d17cbcf41f67399007ad/dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt#L453-L454) API was to avoid creating separate RUM sessions for each background wake-up and impacting user billing. Won't be regression?

Reply:
Indeed it might be a slight regression. I'll try using the `RunningAppProcessInfo` trick (it seems to be working fine)